lib/recipes-data/src/lib/transform.ts

```diff
@@ -69,6 +69,15 @@ export const addSponsorsTransform: (sponsors: Sponsorship[]) => RecipeTransforma
   })
 }
 
+export const addRecipeDatesTransform: (recipeDates: RecipeDates) => RecipeTransformationFunction = recipeDates => {
+  return (recipeData) => ({
+    ...recipeData,
+    ...(recipeDates.lastModifiedDate && { lastModifiedDate: recipeDates.lastModifiedDate.toISOString() }),
+    ...(recipeDates.firstPublishedDate && { firstPublishedDate: recipeDates.firstPublishedDate.toISOString() }),
+    ...(recipeDates.publishedDate && { publishedDate: recipeDates.publishedDate.toISOString() }),
```

Comment:
Little bit niggly as I found this hard to read, probably be easier to simply put it as:

```typescript
return (recipeDate) => ({
   ...recipeData,
   lastModifiedDate: recipeDates.lastModifiedDate ? recipeDates.lastModifiedDate.toISOString() : undefined,
   firstPublishedDate: recipeDates.firstPublishedDate ? recipeDates.firstPublishedDate.toISOString() : undefined,
```
etc/